app/models/shipit/pull_request.rb

```diff
@@ -0,0 +1,184 @@
+module Shipit
+  class PullRequest < ApplicationRecord
```

Comment:
Do we keep track of all of them? Or just the ones that were requested to be merged?